addon/components/frost-action-bar.js

```diff
@@ -80,25 +93,162 @@ export default Component.extend({
   },
 
   @readOnly
-  @computed('controls')
-  isControlsObject (controls) {
-    return typeOf(controls) === 'object'
+  @computed('controls', 'selectedItems.[]', 'moreActions')
+  /**
+   * gets controls for the action bar
+   * @param {Ember.Component[]|object} controls - array or hash of controls
+   * @param {Ember.Object[]|object[]} selectedItems - array of selected items for filtering controls
+   * @param {Ember.Component[]|boolean|object[]|object} moreActions - array or hash of extra controls
+   *  or flag indicating whether to generate them automatically
+   * @returns {Ember.Component[]} array of controls
+   */
+  _controls (controls, selectedItems, moreActions) {
+    controls = this.getNormalizedControls(controls, selectedItems)
+
+    // slice off extra buttons if needed (default)
+    if (moreActions === true) {
```

Comment:
sorry comment is not helping me understand this code

isn't `buttonCount` here giving you count of control items which are NOT `frost-button`s?